packages/react-native/ReactCommon/react/renderer/graphics/LinearGradient.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <vector>
+#include <variant>
+#include <string>
+#include <react/renderer/graphics/Float.h>
+
+namespace facebook::react {
+
+enum class GradientOrientationType {
+  Angle,
+  Direction
+};
+
+struct GradientOrientation {
+  GradientOrientationType type;
+  std::variant<Float, std::string> value;
```

Comment:
Can we parse the `Direction`/`Keyword` version into enum as early as possible instead of keeping string around?